catt/controllers.py

```diff
@@ -143,30 +143,45 @@ def __init__(self, device_name):
         except (pychromecast.error.ChromecastConnectionError, ValueError):
             self.cast = get_chromecast(device_name)
             cache.set(self.cast.name, self.cast.host)
-        time.sleep(0.2)
+
+        self.cast.wait()
+
+    def _check_state(self):
+        if self.cast.app_id == "E8C28D3C" or not self.cast.app_id:
+            raise CattCastError("Chromecast is inactive.")
+        self.cast.media_controller.block_until_active(1.0)
+        if self.cast.media_controller.status.player_state in ["UNKNOWN", "IDLE"]:
+            raise CattCastError("Nothing is currently playing.")
 
     def play_media(self, url, content_type="video/mp4"):
         self.cast.play_media(url, content_type)
+        self.cast.media_controller.block_until_active()
 
     def play(self):
+        self._check_state()
```

Comment:
I'm not in love with calling `_check_state()` in every command. Maybe run it in the constructor and add a `no_state_check` parameter for when we don't want it? That way we only need to specify that parameter once, and the state checks are removed from all the other calls, and we can still use `self.seek`.